src/content/docs/guides/networking/2-restful-api-call.mdx

```diff
@@ -38,103 +38,289 @@ In this first example we make use of a simple GET operation to retrieve a single
+SplashKit.WriteLine(response);
+```
+
+  </TabItem>
+  <TabItem label="Python">
+
+```python
+from splashkit import *
+
+# Get a list of a JSON web resource
+get_data = http_get("https://jsonplaceholder.typicode.com/posts/", 443)
+response = http_response_to_string(get_data)
+free_response(get_data)
+
+# To access each JSON key value pair the string should be split to an vector<json>
+# objects for simplicity sake we output just the string here.
+print(response)
+
+```
 
   </TabItem>
 </Tabs>
 
+After running this example, you'll get an output like this to the terminal:
+
+```bash
+[
+  {
+    "userId": 1,
+    "id": 1,
+    "title": "sunt aut facere repellat provident occaecati excepturi optio reprehenderit",
+    "body": "quia et suscipit\nsuscipit recusandae consequuntur expedita et cum\nreprehenderit molestiae ut ut quas totam\nnostrum rerum est autem sunt rem eveniet architecto"
+  },
+  {
+    "userId": 1,
+    "id": 2,
+    "title": "qui est esse",
+    "body": "est rerum tempore vitae\nsequi sint nihil reprehenderit dolor beatae ea dolores neque\nfugiat blanditiis voluptate porro vel nihil molestiae ut reiciendis\nqui aperiam non debitis possimus qui neque nisi nulla"
+  },
+  ...
+]
+```
+
+Although, your output will include all 100 posts.
+
 ## Example 3: POST Resource
 
 In this example we create a resource on the remote server by using the POST operation. This request requires the header definition specified the content type and the charset. This header must be within a vector as specified in the SplashKit documentation. The data is formed as a JSON object that is converted to a string for the purpose of the HTTP request. Note that the body only contains three parameters as the id is generated on the server.
 
 <Tabs syncKey="code-language">
   <TabItem label="C++">
 
-  ```cpp
-  #include "splashkit.h"
+```cpp
+#include "splashkit.h"
+
+int main()
+{
+    http_response get_data;
+    string response;
+
+    // Create a JSON Web resource
+    // Create the JSON body for the http post call
+    json body = create_json();
+    json_set_string(body, "title", "foo");
+    json_set_string(body, "body", "test entry");
+    json_set_number(body, "userId", 1);
+    // Create the header for the http post call
+    vector<string> headers;
+    headers.push_back("Content-type: application/json; charset=UTF-8");
+    // Send the request
+    get_data = http_post("https://jsonplaceholder.typicode.com/posts", 443, json_to_string(body), headers);
+    response = http_response_to_string(get_data);
+    free_response(get_data);
+    // Output the response
+    json json_response = json_from_string(response);
+    write_line("Following resource has been created");
+    write_line("UserID => " + to_string(json_read_number_as_int(json_response, "userId")));
+    write_line("ID     => " + to_string(json_read_number_as_int(json_response, "id")));
+    write_line("Title  => " + json_read_string(json_response, "title"));
```

Comment:
"to_string" need to be fixed. It can be changed to "std::to_string" in both the lines 239,240.

Reply:
I've just updated this, thank you for picking that up!